e2e/cypress/integration/ui/settings/project_mgmt.spec.ts

```diff
@@ -252,35 +273,8 @@ describe('project management', () => {
         // otherwise, check that the projectID is no longer in the table
       } else {
         cy.get('app-project-list chef-tbody chef-td')
-          .contains(projectID).should('not.exist');
+          .contains(customWithPolsProjectID).should('not.exist');
       }
     });
   });
-
-  // these tests are currently interdependent so mark as flaky until the above isn't
-  itFlaky('can create a project with a custom ID', () => {
```

Comment:
moved this up with the other `create` tests and changed the ID so we wouldn't run into conflicts